credsweeper/common/keyword_checklist.py

```diff
@@ -13,8 +13,9 @@ class KeywordChecklist:
 
     def __init__(self) -> None:
         # used suggested text read style. split() is preferred because it strips 0x0A on end the file
-        with open(self.KEYWORD_PATH, 'r') as f:
-            self.__keyword_set = set(f.read().split())
+        self.__keyword_set = set(self.KEYWORD_PATH.read_text().split())
+        # optimized separately list with morphemes - only substring which may be separated words or 4-chars ending
```

Comment:
```suggestion
        # optimized separately list with morphemes - only substring which may be separated words or 4-chars ending
```
The list of morphemes which can be combined to form words.
The value is considered as variable if at least two of them are present.

Reply:
OK. please, use properly indent, split to 2 lines (limit is 120)